CONTRIBUTING.md

```diff
@@ -31,6 +31,10 @@ Git has the `-s` option (lower case) for `commit` that can sign off a commit for
 
 In order to maintain a high software quality standard, we strongly prefer contributions to follow these rules:
 
+- Each commit in a pull request should be atomic, in the sense that it passes all checks (build, static code analysis,
```

Comment:
Would it make sense to mention that the commit should be atomic in that sense that it is doing one thing at a time, while still being self-contained?

Reply:
Done.

Reply:
Thank you!